notebooks/community/ml_ops/stage7/model_monitoring_custom.ipynb

```diff
@@ -0,0 +1,1267 @@
+{
```

Comment:
<p>Is this a solution for the wait for the email notification? if yes, can we move it above "the bigquery dataset and table..."? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/890/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>